closed/autoconf/toolchain-win.cmake.in

```diff
@@ -35,6 +35,7 @@ if("@OPENJDK_TARGET_CPU_BITS@" EQUAL "64")
 else()
 	set(CMAKE_ASM_MASM_COMPILER "${tool_dir}/ml" CACHE FILEPATH "")
 endif()
+set(CMAKE_ASM_NASM_COMPILER "${tool_dir}/nasm" CACHE FILEPATH "")
```

Comment:
Why don't we want to let nasm be found automatically by [1]?

[1] https://github.com/Kitware/CMake/blob/master/Modules/CMakeDetermineASM_NASMCompiler.cmake#L9-L25

Reply:
Because we are using the windows version of nasm, but the cygwin version of cmake, so cmake will attempt to feed it cygwin style paths which the windows nasm can't parse. 

> Why is this not also required for other java versions?

It is. I just wanted to address review comments here before opening the other PRs

Reply:
Have we tried the native version of cmake instead of [advising](https://github.com/eclipse/openj9/blob/master/doc/build-instructions/cmake.md) to do otherwise?

Reply:
Yes, although I don't remember exactly what the problem was, but I remember it was a big one